release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -561,6 +561,16 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-ebpf-manager-operator-technology-preview_{context}"]
+==== eBPF manager Operator (Tech Preview)
+
+The eBPF manager Operator, available as a Technology Preview, allows you to securely deploy and manage eBPF programs. It facilitates the secure loading, unloading, modifying, and monitoring of eBPF programs in {product-title} clusters. For more information on deploying the bpfman Operator, see xref:../networking/network_security/bpfman-operator/bpfman-operator-about.adoc#bpfman-operator-about[About the eBPF manager Operator].
```

Comment:
I get a 'page not found' error for the link for 'About the eBPF manager Operator.

Reply:
@tmalove, yeah, I need to merge that feature PR, and then this should work.